river/sync.go

```diff
@@ -248,6 +251,19 @@ func (r *River) makeInsertReqData(req *elastic.BulkRequest, rule *Rule, values [
 	}
 }
 
+func CheckFilter(rule *Rule, field string) bool {
+	if rule.Fileter == nil {
+		return true
+	}
+
+	for _, f := range rule.Fileter {
+		if f == field {
```

Comment:
I think there is no need support RegExp . Because there are few similar in the data field. Support for RegExp will reduce the efficiency